tests/unit-tests/test_utils.go

```diff
@@ -23,6 +24,14 @@ const (
 		"}"
 )
 
+var (
+	EmptyTimestamp = &timestamppb.Timestamp{
```

Comment:
Do we still need this?

Reply:
No, we don't need.

Reply:
It needs in tests/unit-tests/did_dic_metadata_test.go.